.github/workflows/non-native.yml

```diff
@@ -224,6 +224,80 @@ jobs:
             time gmake -j3 examples
             echo '::endgroup::'
 
+  android:
+    name: "Android ${{ matrix.platform }} (${{ matrix.build == 'cmake' && 'CM' || 'AM' }}, arm64)"
```

Comment:
Maybe we use vcpkg as well on android? we can bring easily 3rd libs.

Reply:
Yes, we can try that. I can't see other options besides building things manually.

What about BoringSSL + libpsl?

Reply:
I think it possible to do all the libs that we do on windows, do it also on android with vcpkg.
Do you want me to create a similar ci that vcpkg with android? 
I can start with cmake only, and then you can extended it to autoconf.

Reply:
I don't think we need all of those as in Windows. Here we can't run tests, just testing builds and those are quit similar to Linux. We also want to keep CI turnaround fast.

I suggest laying down the framework for it and do a single, useful combination, as suggested above. We can take it from there if there may be more deps that make sense for many Android uses.

Possibly those deps can be interesting, that do not work or missing on Windows, and which we don't have coverage in GHA/linux. (BoringSSL e.g)